lib/streamlit/elements/image_proto.py

```diff
@@ -227,5 +236,8 @@ def marshall_images(
             data = image
 
         (data, mimetype) = _normalize_to_bytes(data, width, format)
-        this_file = media_file_manager.add(data, mimetype=mimetype)
+        this_file = media_file_manager.add(
+            data, mimetype, "%s-%i" % (coordinates, coord_suffix)
+        )
```

Comment:
Do you have a test for this somewhere? You could just have do a test of `st.image` with an image list and then check that a mocked `media_file_manager`'s `add()` method was called with the right inputs.

Reply:
There is, in the code that tests st.image, but i can add a test to the MFM's tests.